ledger/src/blockstore_db.rs

```diff
@@ -141,7 +141,15 @@ pub mod columns {
     pub struct SlotMeta;
 
     #[derive(Debug)]
-    /// The orphans column
+    /// The orphans column.
+    ///
+    /// This column family tracks whether a slot has a parent.  If a slot does
+    /// not have a parent yet, it is by definition an orphan slot and will have
+    /// an entry in this column family with true value indicating it is an
+    /// orphan.
+    ///
+    /// Note that NOT_FOUND or an entry with false value indicates this slot
+    /// is not an orphan slot (i.e., has a parent slot.)
```

Comment:
This is incorrect - we delete the entry when the slot gains a parent. Added a note about this in my above suggestion.

```rust
        // At this point this slot has received a parent, so it's no longer an orphan
        if was_orphan_slot {
            write_batch.delete::<cf::Orphans>(slot)?;
        }
```